Sources/Defaults/Defaults.swift

```diff
@@ -36,6 +36,8 @@ public enum Defaults {
 }
 
 public typealias _Defaults = Defaults
+
+@available(macOS 11, iOS 14, tvOS 14, watchOS 7, *)
```

Comment:
These are not needed.

https://github.com/sindresorhus/Defaults/commit/fcdf1967b967840862cd7a5bde5712b45bf79493